portfolio/src/main/webapp/randomBook.html

```diff
@@ -13,5 +15,20 @@ <h1>Orpaz's Favorites Books</h1>
       <button onclick="randomizeImage()">book</button>
       <div id="random-image-container"></div>
     </div>
+    <div id="chart-container"> </div>
```

Comment:
This is duplicating data you currently have server-side. Can you avoid this duplication somehow?
E.g. - add another servlet that will retrieve the full list, and use it to populate this form?
Or just add the "vote" button to the response of displayRandomBook (to allow voting on the specific book)?